src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMA = ",";
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile;
+        File file = new File(fromFileName);
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
Consider using try-with-resources statement to automatically close BufferedReader